conf/application.conf

```diff
@@ -80,6 +80,11 @@ microservice {
       host = localhost
       port = 9570
     }
+
+    essttp-backend {
+      host = localhost
+      port = 9156
+    }
```

Comment:
Have we got equivalent config set up in app-config-base?
Do we need an update to our service manger config profile?

Reply:
Here's config-base changes: https://github.com/hmrc/app-config-base/pull/9797
And the service manager profile: https://github.com/hmrc/service-manager-config/pull/6105